gui/autotests/tst_gui.cpp

```diff
@@ -0,0 +1,31 @@
+#include <QCoreApplication>
```

Comment:
One more comment, please rename this to be consistent with other tests - eg. `gui_test.cpp`.